zcash_test_vectors/transaction_v6.py

```diff
@@ -81,6 +81,38 @@ def __bytes__(self):
         return ret
 
 
+class ActionGroupDescription(object):
+    def __init__(self, rand, anchor_orchard, proofs_orchard, is_coinbase):
+        self.vActionsOrchard = []
+        # There must always be a non-zero number of Action Descriptions in an Action Group.
+        for _ in range(rand.u8() % 4 + 1):
+            self.vActionsOrchard.append(OrchardZSAActionDescription(rand))
+        self.flagsOrchard = rand.u8()
+        # Three flag bits are defined, we set enableZSA to true.
+        self.flagsOrchard = (self.flagsOrchard & 7) | 4
+        if is_coinbase:
+            # set enableSpendsOrchard = 0
+            self.flagsOrchard &= 2
+        self.anchorOrchard = anchor_orchard
+        self.proofsOrchard = proofs_orchard
+        self.nAGExpiryHeight = 0
+
+    def __bytes__(self):
+        ret = b''
+        ret += write_compact_size(len(self.vActionsOrchard))
+        for desc in self.vActionsOrchard:
```

Comment:
why `desc`? should be `a` or `action` (x2)

Reply:
This follows the convention upstream (see [here](https://github.com/zcash/zcash-test-vectors/blob/1ac6808080e302ba2c7b02333d6fa3713f7d1f0e/zcash_test_vectors/transaction.py#L546)). `desc` for Action **desc**ription.

Reply:
I see